specification/azsadmin/resource-manager/fabric/readme.md

```diff
@@ -147,54 +146,3 @@ csharp:
   output-folder: $(csharp-sdks-folder)/Generated
   clear-output-folder: true
 ```
-
```

Comment:
Hi, why do we remove this part? Should we bring it back too?

Reply:
This part is added in readme.azsautogen.md for autorest generation config.

Reply:
I think this one is used for autorest v3?